### PR TITLE
Calls: update FAQ with more information and IP addresses

### PR DESCRIPTION
**Pull request by renandincer**

### CODE REVIEW

File: content/calls/turn/faq.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ TURN protocol, [RFC 8656](https://datatracker.ietf.org/doc/html/rfc8656), does n
 
 Cloudflare Calls TURN server runs on [Cloudflare's global network](https://www.cloudflare.com/network) - a growing global network of thousands of machines distributed across hundreds of locations, with the notable exception of the Cloudflare's [China Network](/china-network/).
 
+### Does Cloudflare Calls TURN use the Cloudflare Backbone or is there any "magic" Cloudflare do to speed connection up?
+
+Cloudflare Calls TURN allocations are homed in the nearest available Cloudflare datacenter to the TURN client via anycast routing. If both ends of a connection is using Cloudflare Calls TURN, Cloudflare will be able to control the routing and, if possible, route TURN packets through the Cloudflare backbone.
```
Comment (dcpena) [inline content/calls/turn/faq.md]:
```suggestion
Cloudflare Calls TURN allocations are homed in the nearest available Cloudflare datacenter to the TURN client via anycast routing. If both ends of a connectionare using Cloudflare Calls TURN, Cloudflare will be able to control the routing and, if possible, route TURN packets through the Cloudflare backbone.
```

File: content/calls/turn/faq.md [hunk 2]
```diff
@@ -38,15 +42,25 @@ There is no performance or feature level difference for Cloudflare Calls TURN se
...
+
+Please allowlist the following IP addresses:
+- `2a06:98c1:3200::1/128`
+- `141.101.90.1/32`
+
+Cloudflare tries to, but cannot guarantee that the IP addresses used for the TURN service won't change unless this is in your enterprise contract. For more details about static IPs, guarantees and other arrangements please discuss with your enterprise account team. Your enterprise account will be able to provide additional addresses to allowlist as future backup to achieve address diversity.
 
-Please allowlist Cloudflare's published [IP address ranges](https://www.cloudflare.com/ips/). Cloudflare's TURN service will use an IP address from this list. For more details about static IPs, guarantees and other arrangements please discuss with your enterprise account team.
+If you are allowlisting IP addresses, and don't have a enterprise contract with different instructions, you must set up alerting that detects changes the DNS response from `turn.cloudflare.com` (A and AAAA records) and update the hardcoded IP address(es) accordingly within 14 days of the DNS change.
```
Comment (dcpena) [inline content/calls/turn/faq.md]:
```suggestion
If you are allowlisting IP addresses and do not have a enterprise contract with different instructions, you must set up alerting that detects changes the DNS response from `turn.cloudflare.com` (A and AAAA records) and update the hardcoded IP addresses accordingly within 14 days of the DNS change.
```

**Review by dcpena — APPROVED** [review]

File: content/calls/turn/faq.md (hunk 1 above)
Comment (renandincer) [inline content/calls/turn/faq.md]:
```suggestion
Cloudflare Calls TURN allocations are homed in the nearest available Cloudflare datacenter to the TURN client via anycast routing. If both ends of a connection are using Cloudflare Calls TURN, Cloudflare will be able to control the routing and, if possible, route TURN packets through the Cloudflare backbone.
```

**Review by dcpena — APPROVED** [review]

**Comment by nils-ohlmeier** [comment]
The TURN overview documentation still points to the generic Cloudflare list of all IP addresses. Probably worth updating the two references in there as well to avoid confusion.

**Review by nils-ohlmeier — APPROVED** [review]

**Pull request merged**